CODE_OF_CONDUCT.md

```diff
@@ -57,9 +57,8 @@ further defined and clarified by project maintainers.
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
 reported by contacting the project team:
 
-- lauren.eastridge@formidable.com
-- phil.pluckthun@formidable.com
-- Twitter DMs: @_philpl
+- phil@0no.co
+- grant.sander@formidable.com
```

Comment:
Feel free to add me here as well with jovi@preact.dev or decroockjovi@gmail.com

Reply:
done! 🙌